tests/nlu/test_evaluation.py

```diff
@@ -1219,3 +1219,25 @@ def test_replacing_fallback_intent():
 def test_is_entity_extractor_present(components, expected_result):
     interpreter = Interpreter(components, context=None)
     assert is_entity_extractor_present(interpreter) == expected_result
+
+
+def test_plot_attribute_confidences_with_invalid_values(tmp_path: Path):
+    path = tmp_path / "evaluation"
+    path.mkdir()
+    report_folder = str(path / "reports")
+
+    rasa.shared.utils.io.create_directory(report_folder)
+
+    intent_results = [
+        IntentEvaluationResult("", "restaurant_search", 76, -21),
+        IntentEvaluationResult("greet", "greet", -32, 99),
```

Comment:
@wochinge I put some random integer values here and this gives me a `pos_hist = [99]` [here](https://github.com/RasaHQ/rasa/blob/1aa1b6c93ba2603064737814f9fcdce764af40d9/rasa/nlu/test.py#L329) so the exception is raised correctly and the test pass. Maybe it is a stupid question but I was just wondering if I will always get the same result? It looks deterministic but I still wanted to make sure and thus ask.